packages/in_app_purchase/lib/src/in_app_purchase_connection/google_play_connection.dart

```diff
@@ -34,6 +34,51 @@ class GooglePlayConnection
     return _billingClient.isReady();
   }
 
+  @override
+  Future<QueryPastPurchaseResponse> queryPastPurchases(
+      {String applicationUserName}) async {
+    List<PurchasesResultWrapper> responses = await Future.wait([
+      _billingClient.queryPurchaseHistory(SkuType.inapp),
+      _billingClient.queryPurchaseHistory(SkuType.subs)
+    ]);
+
+    BillingResponse errorInApp = responses.first.responseCode;
+    BillingResponse errorSubs = responses.last.responseCode;
+    String errorMessage = null;
+    if (errorInApp != BillingResponse.ok) {
+      errorMessage = errorInApp.toString();
+    }
+    if (errorSubs != BillingResponse.ok && errorSubs != errorInApp) {
+      errorMessage += ', ${errorSubs.toString()}';
+    }
+
+    List<PurchaseDetails> pastPurchases;
+    if (responses == null) {
```

Comment:
I don't think `responses` could ever be null here since we're initializing it as an array of Futures and then not reassigning it. Do we need this block?